### DISCUSSION TITLE
Remove spellcheck for smaller apk size (prevent it from being compiled)

### DISCUSSION THREAD

**Discussion by sabzo** [florisboard/florisboard#1269] [Q&A]
@patrickgold 

What would be the safest/easiest way to disable the spellchecker? In my need I don't particularly need a spellchecker and would like to be able to drop in a custom AI spellchecker.

Thanks!

**Comment by MXC48-zz** [comment]
> @patrickgold 
> 
> What would be the safest/easiest way to disable the spellchecker? In my need I don't particularly need a spellchecker and would like to be able to drop in a custom AI spellchecker.
> 
> Thanks!

In the phone settings go to keyboard and then to spell check and disable the spell check option

**Reply by sabzo** [reply]
@MXC48 I mean to actually remove the code from being compiled in order to have a smaller apk size, not to disable it once the spellchecker is installed, but to prevent the spellchecker C and related kotlin from being compiled in the first place.

**Reply by MXC48-zz** [reply]
> @MXC48 I mean to actually remove the code from being compiled in order to have a smaller apk size, not to disable it once the spellchecker is installed, but to prevent the spellchecker C and related kotlin from being compiled in the first place. 

In this case I can't help, sorry
